externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Conversion/Passes.td

```diff
@@ -13,6 +13,28 @@
 include "mlir/Pass/PassBase.td"
 
 #ifdef TORCH_MLIR_DIALECTS_ENABLE_TCP
+
+#ifdef TORCH_MLIR_ENABLE_STABLEHLO
+
+//===----------------------------------------------------------------------===//
+// StablehloToTcp
+//===----------------------------------------------------------------------===//
+
+def ConvertStablehloToTcp
```

Comment:
Very minor nit: should this be `ConvertStableHloToTcp`?

Reply:
We've been using "StableHLO" as the name of the dialect in prose, and "Stablehlo" or "stablehlo" as the name in code (e.g. `class StablehloDialect` in [StablehloOps.h](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.h#L52)). Let's use this convention in this repository as well?